--- xdelta3.h.orig
+++ xdelta3.h
@@ -167,7 +167,7 @@
 #endif
 
 /* SIZEOF_SIZE_T should be defined in config.h */
-static_assert(SIZEOF_SIZE_T == sizeof(size_t), "SIZEOF_SIZE_T not correctly set");
+/* static_assert(SIZEOF_SIZE_T == sizeof(size_t), "SIZEOF_SIZE_T not correctly set"); */
 
 /* The code relies on USIZE_T_MAX == XD3_INVALID_OFFSET for overflow
  * checks, although it's not entirely necessary.  Just be careful if
